node/DEVELOPER.md

```diff
@@ -27,9 +27,19 @@ Software Dependencies
 
 ```bash
 sudo apt update -y
-sudo apt install -y nodejs npm git gcc pkg-config protobuf-compiler openssl libssl-dev
```

Comment:
No need to install nvm for specific version
Use `curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -` before the existing apt install
Then use the same existing install.
Nvm is amazing for switching version easily, but its not necessary if you need just one specific v, and we support 16.

Reply:
You might have incompatibility between the Node and the npm.
So you better have the ability to move fast between the versions.

Reply:
I think this note is enough:
https://github.com/aws/glide-for-redis/blob/b08f5e8476bbb15c90c9ea9cafadeace3790727a/node/DEVELOPER.md?plain=1#L15
We want to avoid suggesting that the user must install the latest Node version